### DISCUSSION TITLE
Discuss what kind of accessibility we can support.

### DISCUSSION THREAD

**Discussion by korgan00** [Qiskit/qiskit.org#2372] [General]
After working on different accessibility issues and researching them, we can improve many things in these terms.
There are many accessibility levels we could support, but it implies development time, and we should prioritize.

Types of accessibility:
 - Colorblind.
 - Blind
 - Deaf
 - Keyboard only
 - Photosensitive epilepsy (Videos, gifs, and animations)

I accept the suggestions of accessibility types I could miss.
My intention here is to discuss it.
I also suggest creating a new label called `type: accessibility`.

**Comment by techtolentino** [comment]
For reference, I'm adding the link to the [IBM Accessibility Checklist](https://www.ibm.com/able/guidelines/ci162/accessibility_checklist.html), which we can use for guidance.

**Comment by eddybrando** [comment]
While I strongly believe that we should strive to make our website accessible to everyone despite any kind of impediment, I also agree with the fact that the development efforts need to be properly managed.

It's quite hard to know what to prioritise exactly without just guessing what accessibility needs our users are more likely to have, since I believe we don't have statistics about this.

From experience, I'd probably try to start with some easy to assess quick wins, for instance, by running the Lighthouse accessibility check on every page and trying to solve the issues reported.

After that I'd try to tackle the keyboard navigation, since that's something that I believe may affect a big percentage of users with and without accessibility needs.

Then I'd try to check and improve the screen reader usage, at least to a point where the website is understandable and there are no critical issues, like loops that prevent further navigation.

I think it should be an ongoing effort to check and try to improve on accessibility whenever we work on an issue. But it's also true that we need some more in-depth reports about the current state of our website and what we need to do.

---

Regarding the label, `accessibility` is now available.

**Comment by eddybrando** [comment]
A more concrete suggestion: We can start by trying to conform to all Level A WCAG 2.1 points listed in this reference: https://www.w3.org/WAI/WCAG21/quickref/

**Comment by JRussellHuffman** [comment]
If the conclusion of this issue includes things that need design support and currently aren't included, let me know. happy to update the design files and to keep them in consideration for future page designs.

**Comment by JRussellHuffman** [comment]
This discussion came up in my email again (exactly one year later apparently). Is this something we should start thinking about again? We could always put out a survey to directly ask about accessibility needs. I also wonder if any of our analytics tools can provide some information about accessibility needs (like % of users who use screen readers).

**Reply by eddybrando** [reply]
I believe that we should definitely bring up the accessibility improvements topic again, but at this point I don't think that we need a discussion about what kind of accessibility support we can provide, but rather just try and implement some assessment tool to give us a very basic and general idea of how we're currently doing with the most basic stuff, and try to improve on those metrics.

After we cover the basics, I think we should think about specifics. But we should first start measuring our current state.

**Reply by JRussellHuffman** [reply]
I think that's a great idea. Let's get some data about what we need before we start implementing things.
